inline_scan.sh

```diff
@@ -8,8 +8,8 @@ set -eou pipefail
 
 # If using a locally built stateless CI container, export SYSDIG_CI_IMAGE=<image_name>.
 # This will override the image name from Dockerhub.
-INLINE_SCAN_IMAGE="${SYSDIG_CI_IMAGE:-docker.io/anchore/inline-scan:v0.6.1}"
 DOCKER_NAME="${RANDOM:-temp}-inline-anchore-engine"
+INLINE_SCAN_IMAGE="${INLINE_SCAN_IMAGE:-}"
```

Comment:
I'm thinking to use an environment variable (and not a flag) for two reasons:

1) There are already plenty of flags
2) This customisation should be done as last resort or for testing purposes